wrangler-transform/docs/Wrangler-transform.md

```diff
@@ -28,6 +30,12 @@ not configured as part of the output schema, and you are using the `set columns`
 directive, you may see inconsistent behavior. Use the `drop` directive to drop any fields
 that are not used in the data prep.
 
+Precondition language is selected to `JEXL` by default. It can be switched between `SQL`
+and `JEXL`.
+
+If precondition language is selected as `SQL`, the directive and UDD fields throw error
```

Comment:
Rephrase to: "the directive and UDD fields throw *a validation* error *if they are non-empty*".